src/cellfinder_core/detect/filters/volume/structure_splitting.py

```diff
@@ -157,7 +161,7 @@ def split_cells(
     # centres is a list of lists of centres (1 list of centres per ball run)
     ns, centres = iterative_ball_filter(vol)
     ns.insert(0, 1)
-    centres.insert(0, [relative_orig_centre])
+    centres.insert(0, relative_orig_centre)
```

Comment:
This is now not a "list of lists of centres" as the comment above on line 161 says it should be - given the return of `iterative_ball_filter` hasn't changed, I think this *should* stay as `[relative_orig_centre]`?

I am guessing you changed this for a reason though... any thoughts?

Reply:
AFAICT the return of `iterative_ball_filter` did change from `List[List[Point]]` to `[List[np.ndarray]`, so `mypy` was complaining about the typing with the previous `[relative_orig_centre]`, which was the reason I removed it.

However, I hadn't thought things through, and therefore this is still a good catch of a bug I introduced! I've adapted the comment and fixed the bug to reflect my intentions now:
```python
# centres is a list of arrays of centres (1 array of centres per ball run)
ns, centres = iterative_ball_filter(vol)
ns.insert(0, 1)
centres.insert(0, np.array([relative_orig_centre]))
```